bokeh/server/server.py

```diff
@@ -128,7 +134,7 @@ def __init__(self, applications, io_loop=None, **kwargs):
             tornado_kwargs['redirect_root'] = kwargs.get('redirect_root', True)
 
             self._tornado = BokehTornado(self._applications, self.prefix, **tornado_kwargs)
-            self._http = HTTPServer(self._tornado, xheaders=kwargs.get('use_xheaders', False))
+            self._http = HTTPServer(self._tornado, xheaders=kwargs.get('use_xheaders', False), **tornado_server_kwargs)
```

Comment:
There is a slight chance for user error here. If someone puts `xheaders` in `tornado_server_kwargs` there will be a duplicate kwarg passed to `HTTPServer`. I think we can be slightly more careful here. How about:
```
tornado_server_kwargs.setdefault('xheaders', kwargs.get('use_xheaders', False))
self._http = HTTPServer(self._tornado, **tornado_server_kwargs)
```

This should have any `xheaders` value already in `tornado_server_kwargs` (which should only really happen if we are *not* running in the from the command line) take precedence, otherwise use the `use_xheaders` or default to `False`